include/aotriton/_internal/triton_kernel.h

```diff
@@ -24,11 +27,24 @@ class TritonKernel {
   void clear_decompressed_image();
 #endif
 private:
+  std::tuple<hipFunction_t, hipError_t> load_for_device(int device_id, const char* kernel_name);
+  hipFunction_t cfind_function(int device_id) const;
+
   const void* kernel_image_ = nullptr;
   size_t image_size_ = 0;
+  struct DeviceFunction {
+    DeviceFunction(int device_id_,
+                   hipModule_t mod_,
+                   hipFunction_t func_);
+    ~DeviceFunction();
+    int device_id = -1;
+    hipModule_t mod = nullptr;
+    hipFunction_t func = nullptr; 
+  };
+  std::unordered_map<int, DeviceFunction> funcache_;
```

Comment:
funcache_ --> fun_cache_, just to be consistent with your own coding style

Reply:
Oh this is a similar to trick of "AOT Triton -> AOTriton", and "Func cache -> Funcache"